routes/api.py

```diff
@@ -235,44 +239,71 @@ def feedback(version: int = 1) -> Any:
     return better_jsonify(ok=True)
 
 
+OPTIONS_FLAGS = {
+    "annotate_unparsed_sentences": "bool",
+    "suppress_suggestions": "bool",
+    "ignore_wordlist": "list",
+    "ignore_rules": "list",
+}
+
+
+def opts_from_request(rq: Request) -> Dict[str, Any]:
+    """Extract valid options from a request into dict."""
+    d = dict()
```

Comment:
Annotate: d: Dict[str, Any] = dict()

...and strictly speaking, the annotation could (and should?) be made accurate:

`OptionsDict = Dict[str, Union[bool, List[str]]]`